pages/common/xargs.md

```diff
@@ -10,6 +10,10 @@
 
 `{{arguments_null_terminated}} | xargs -0 {{command}}`
 
-- example: list unneeded packages with deborphan and remove them with apt-get
+- example: delete all files that start with 'M'
```

Comment:
It might be helpful. Currently, some of the commands in existing tldr pages are just man page entries, retyped. In other words, they don't show example uses, just `command {{token}}`. 

[An example simple commands](https://github.com/tldr-pages/tldr/blob/master/pages/common/sort.md)

[An example of an example](https://github.com/tldr-pages/tldr/blob/master/pages/common/nmap.md)

I don't feel strongly about it, and I can remove it. It's easier than going back and distinguishing between real-world  examples and overly-simple commands. Though it might be worth mentioning the difference and outlining which is desired in the CONTRIBUTING document